stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/ClientState.kt

```diff
@@ -110,30 +110,34 @@ class ClientState(client: StreamVideo) {
         removeRingingCall()
     }
 
-    fun addRingingCall(call: Call) {
-        // TODO: behaviour if you are already in a call
+    fun addRingingCall(call: Call, ringingState: RingingState) {
```

Comment:
I am starting `CallService` in this method. I need to know what trigger to send to the service, so I also added a `ringingState` parameter. `addRingingCall` is called in places where we know if the call is incoming or outgoing, so it's ok to add this param?

Reply:
Just as a thought, maybe we need to deprecate `addRingingCall` and use create new API `addIncommingCall` and `addOutgoingCall` call.